tools/default.nix

```diff
@@ -0,0 +1,65 @@
+{ pkgs ? import <nixpkgs> {} , withEditor ? false }:
+
+pkgs.stdenv.mkDerivation rec {
+  name = "openclonk";
+
+  gitRef = pkgs.lib.commitIdFromGitRepo ../.git;
+
+  src = builtins.filterSource (path: type: ! builtins.elem (baseNameOf path) [
+    ".git" # leave out .git as it changes often in ways that do not affect the build
+    "default.nix" # default.nix might change, but the only thing that matters is what it evaluates to, and nix takes care of that
+    "result" # build result is irrelevant
+    "build"
+  ]) ./..;
+
+  enableParallelBuilding = true;
+
+  hardeningDisable = "format";
+
+  nativeBuildInputs = with pkgs; [ cmake ];
+
+  dontStrip = true;
+
+  buildInputs = with pkgs; [
+    SDL2 libvorbis libogg libjpeg libpng freetype glew tinyxml
+    openal freealut pkgconfig
```

Comment:
Shouldn't pkgconfig go into nativeBuildInputs?

When I tried doing this, I also needed SDL_mixer, miniupnpc_2, mesa and xorg.xrandr, and didn't seem to need tinyxml, and the readme says we need zlib but I didn't try that.